pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -1062,6 +1102,24 @@ templating:
 
 jobs:
 
+# Run DB Teardown
+- name: "Reset DB"
+  disable_manual_trigger: true
+  plan:
+  - get: midnight
```

Comment:
Use the `every-midnight` trigger instead

Reply:
Sorry forgot you added one with your card 👍